tests/test_dates.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+import time
+from elifecrossref import dates
+
+
+class TestIsoDateString(unittest.TestCase):
+
+    def test_iso_date_string(self):
+        date_string = '2020-06-03'
+        pub_date = time.strptime(date_string, '%Y-%m-%d')
+        self.assertEqual(dates.iso_date_string(pub_date), date_string)
+
+    def test_iso_date_string_none(self):
+        self.assertIsNone(dates.iso_date_string(None))
```

Comment:
another approach that may help avoid all this boilerplate is:

```python
cases = [
   (None, None),
   ((2019, 12, 31, 0, 0, 0, 1, 365, -1), "2019-12-31"),
   (tuple(), None)
]
for given, expected in cases:
    self.assertEqual(expected, dates.iso_date_string(given))
```

it lets you add new cases quickly including expected error cases (`tuple()` will cause a `TypeError` for example).

suggestion only

Reply:
Hmm, I deleted this `test_dates.py` module now, since it was just testing `iso_date_string()`, and it is simple and not requiring extensive tests.